x-pack/plugins/watcher/public/application/sections/watch_list_page/watch_list_page.tsx

```diff
@@ -446,7 +449,15 @@ export const WatchListPage = () => {
           : '',
     };
 
+    const handleOnChange = ({ queryText, error: queryError }: EuiSearchBarOnChangeArgs) => {
+      if (!queryError) {
```

Comment:
Should we surface an error message if there is an error? Otherwise, it might not be clear to the user why a particular search returned no results.